rios/__init__.py

```diff
@@ -22,8 +22,5 @@
 
 """
 
-from distutils.version import LooseVersion
-
-RIOS_VERSION = '1.4.11'
-RIOS_VERSION_OBJ = LooseVersion(RIOS_VERSION)
```

Comment:
@neilflood do you know of any code that uses `RIOS_VERSION_OBJ`?

Reply:
Good question. No, I don't know of anything. I think I remember putting that in there because it would be a more robust way of comparing version numbers, to ask questions like "if RIOS_VERSION_OBJ >= something:", but I don't think I have ever used it. I am happy enough to drop it, and wear the flak if we break anyone's code. If distutils is disappearing, I don't yet know how one ought to do this sort of comparison, but we can cross that bridge when we get there.